whiteblock/cmd/transactions.go

```diff
@@ -37,9 +38,28 @@ The primary use of these methods is to be able to send one line commands through
 testing script that will be able to automate transaction tests.
 */
 
+var sendTxCmd = &cobra.Command{
+	// Hidden: true,
+	Use:   "send <command>",
+	Short: "Send a single transaction",
+	Long: `
+Send will allow the user to send a transaction 
+
+Usage:
+	whiteblock tx send <command> [flags]
+	whiteblock tx send [command] 
+
+Available Commands:
+	single   Send a single transaction from an account to another account
+	to       Send transaction data to an account
+	`,
+	Run: util.PartialCommand,
+}
+
 var sendSingleTxCmd = &cobra.Command{
 	// Hidden: true,
-	Use:   "send",
+	Use:   "single",
+	Aliases: []string{""},
```

Comment:
Don't put an empty string in an array for Aliases, instead, just don't include it.